pkg/apisix/plugin_metadata.go

```diff
@@ -179,7 +179,7 @@ type pluginMetadataMem struct {
 func newPluginMetadataMem(c *cluster) PluginMetadata {
 	return &pluginMetadataMem{
 		url:      c.baseURL + "/plugin_metadatas",
```

Comment:
```suggestion
		url:      c.baseURL + "/plugin_metadata",
```

Reply:
I saw this [code](https://github.com/apache/apisix/blob/3b16fce060fbac39ee054784213a6bc81df526fc/apisix/control/v1.lua#L468) in the apisix repo, is it related?

Reply:
pls refs: https://apisix.apache.org/docs/apisix/admin-api/#plugin-metadata

Actually, the `plugin_metadata` is the correct one

Reply:
The code you mentioned is the control API
https://apisix.apache.org/docs/apisix/control-api/#get-v1plugin_metadatas.

Reply:
I see, I actually don't quite understand what is the url used for, why in my testing even though the url is supposedly wrong but it's still working ? As you are the original committer of the code could you briefly explain to me for my learning purpose?

Reply:
It only represents simulating the corresponding `plugin_metadata` etcd key here, but in fact, it does not have many uses.